packages/appsync-emulator-serverless/README.md

```diff
@@ -44,6 +44,18 @@ const dynamodb = new DynamoDB({
 const client = new DynamoDB.DocumentClient({ service: dynamodb });
 ```
 
+#### Custom build prefix for webpack, typescript
+For compatibility with plugins such as Serverless Webpack that alllow the usage of webpack
+you will need to add the following configuration to your project's `serverless.yml` file
+
+```
+custom:
+  appsync-offline:
+    port: $APPSYNC_OFFLINE_PORT
```

Comment:
what is `port` used for ?

Reply:
Good point, it sets the port for Appsync offline to run at but unneeded for this example.